.github/workflows/ci.yml

```
@@ -5,12 +5,22 @@ on:
   pull_request:
 jobs:
   test:
-    runs-on: ubuntu-latest
-    name: test (Python ${{ matrix.python-version }})
+    runs-on: ${{ matrix.os }}
+    name: test (Python ${{ matrix.python-version }} on ${{ matrix.os-label }})
     strategy:
+      fail-fast: false
       matrix:
         # keey it sync with tox.ini [gh-actions] section
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
+        os: ["ubuntu-latest"]
+        os-label: ["Ubuntu"]
+        include:
+          - python-version: "3.8"
+            os: "windows-latest"
+            os-label: "Windows"
+          - python-version: "3.8"
+            os: "macos-latest"
+            os-label: "macOS"
```

```suggestion
          - {python-version: "3.8", os: "windows-latest", os-label: "Windows"}
          - {python-version: "3.8", os: "macos-latest", os-label: "macOS"}
```
could be easier to read